src/write/coff/object.rs

```diff
@@ -825,7 +827,5 @@ impl<'a> Object<'a> {
 
 // JamCRC
 fn checksum(data: &[u8]) -> u32 {
-    let mut hasher = crc32fast::Hasher::new_with_initial(0xffff_ffff);
-    hasher.update(data);
-    !hasher.finalize()
+    crc_fast::crc32_iso_hdlc(data)
```

Comment:
This isn't equivalent to the CRC it is replacing, and `CrcAlgorithm::Crc32Jamcrc` didn't work either.